src/test/java/racing/car/PositionTest.java

```diff
@@ -0,0 +1,48 @@
+package racing.car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racing.car.car.Position;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class PositionTest {
+
+    @Test
+    @DisplayName("move 메서드는 3 이상인 경우 position은 증가하지 않는다.")
+    void 앞으로_이동_4이하() {
+        Position position = new Position();
+        position.move(3);
+
+        assertThat(position).extracting("position").isEqualTo(0);
+
+    }
+
+    @Test
+    @DisplayName("move 메서드는 4 이상인 경우 position이 1증가 된다.")
+    void 앞으로_이동_4이상() {
+        Position position = new Position();
+        position.move(4);
+
+        assertThat(position).extracting("position").isEqualTo(1);
+    }
+
+    @Test
+    @DisplayName("Position 클래스의 position 값을 비교하여 같으면 True, 다르면 False")
+    void same_메서드_테스트() {
+        Position position = new Position(1);
+
+        assertThat(position.isSame(1)).isTrue();
+        assertThat(position.isSame(0)).isFalse();
+    }
+
+    @Test
+    @DisplayName("max 최대값 반환 테스트")
+    void max_최대값_테스트() {
+        Position position = new Position(5);
+        int max =position.max(3);
```

Comment:
커밋 전에 사용하지 않는 코드나 포맷팅을 한 번 확인해 보시면 좋을 것 같아요.
IDE의 reformat 기능을 활용해 보시길 추천드립니다.
```suggestion
        int max = position.max(3);
```